playwright-e2e/tests/rgp/adult-enrollment.spec.ts

```diff
@@ -152,5 +164,22 @@ test.describe('Adult Self Enrollment', () => {
     // fields should be disabled. check one field to verify is disabled
     expect(await tellUsAboutYourFamily.yourTitle().isDisabled()).toEqual(true);
     expect(await tellUsAboutYourFamily.yourFirstName().isDisabled()).toEqual(true);
+    //Go to DSM to verify the newly created account can be found there
+    const dsm = new dsmHome(page);
```

Comment:
Recommendation: Make a test to do dsm checking. That way, if DSM checking fails, RGP test is not affected.
```
test('Go to DSM to verify the newly created account can be found @functional @enrollment @rgp')
```